macros/src/main/scala/gestalt/macros/DefMacros.scala

```diff
@@ -20,6 +20,26 @@ object plusObject {
         toolbox.Lit(null)
     }
   }
+
+  inline def varargs(tped: Int*): Int = meta {
+    val q"$items: $_" = tped
+    items match {
+      case toolbox.SeqLiteral(items: Seq[toolbox.Tree]) =>
+        items.reduceLeft((a, b) => q"$a + $b")
+      case _ =>
+        q"$items.reduce((a:Int,b:Int)=> a + b)"
+    }
+  }
```

Comment:
I see making `tpd` as `List[Tree]` will need the expander to do some preprocessing on the argument trees. It introduces some inconvenience to the implementation, but for macro writers, it seems to be a big gain. Or, I may miss something here.